src/AutoRest.CSharp/LowLevel/Output/RestClientOperationMethods.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using AutoRest.CSharp.Common.Input;
+using AutoRest.CSharp.Common.Output.Models;
+using AutoRest.CSharp.Common.Output.Models.Responses;
+using AutoRest.CSharp.Generation.Types;
+using AutoRest.CSharp.Output.Samples.Models;
+
+namespace AutoRest.CSharp.Output.Models
+{
+    internal record RestClientOperationMethods
+    (
+        Method CreateRequest,
+        Method? CreateNextPageMessage,
+        Method? Protocol,
+        Method? ProtocolAsync,
+        Method? Convenience,
+        Method? ConvenienceAsync,
+        Method? NextPageConvenience,
+        Method? NextPageConvenienceAsync,
+        ResponseClassifierType ResponseClassifier,
+
+        int Order,
```

Comment:
since this is a workaround, let us get a comment to mark it.

Reply:
Added a comment